### ISSUE TITLE
0.2.3 opening old style dots on 10.15.7

### ISSUE THREAD

**Issue by greeneryAO** [tonsky/AnyBar#87]
when I launch it, it uses old dot styles, rarely it uses new, but if I try fill and hollow on them which I wanted it doesn't work and goes back to previous version dots, is this version only for big sur?

**Comment by tonsky** [comment]
No, it’s not only for Big Sur. Are you sure you’re using the latest version?

> if I try fill and hollow on them which I wanted it doesn't work and goes back to previous version dots

Can you post exact commands and screenshot of what happens? Thank you!

**Comment by greeneryAO** [comment]
well just as example if I double click the anybar app, newest version 0.2.3 modified on february 19 2021, as shown in right click get info.

Either in its own unzipped downloads folder or move to application folder, it opens up default port with a black circle and fat white outline, which is old and wrong so I know it's somehow using old settings, is there some way I have to uninstall old settings somehow?

I can post vids and pics, but tomorrow, it's late right now where I live, thanks for reply by the way, I could definitely use 2 new indicators as I have used up all colors already.

filled makes it all white, hollow black with fat white outline, question is actually new and black as it should, it's like taking settings from both old and new and mixing them up, strange.

**Comment by greeneryAO** [comment]
Somehow the old version is still on my system even though I deleted it through appcleaner app, and the plist, and installed through brew or just download the new zip, it exists somewhere on my system and is conflicting, I can even get both versions to start at once by running an old startup script as seen in this picture 

https://imgur.com/NeHZCy6

weird, but I have scripts and things set up very complex with anybar so I guess I'll just have the messed up old and new together version that I have.

**Comment by greeneryAO** [comment]
everything opens up as new icons, but filled and hollow do not work and hollow even produces an old black and fat white outline icon which shouldn't even be on my computer because I got rid of all old anybar app, weird but oh well.  Buying a new macbook soon that should work on.
